torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -933,6 +933,7 @@ class OutlierDetector(DetectorBase):
 
     Args:
         ratio_threshold (float, optional): The threshold for p_r to determine if there are outliers in activations
+            Should be >= 1
```

Comment:
would be good to add that statistical threshhold as a parameter here potentially in this PR or a later one

Reply:
will add in a future PR